rust/schema/annotations.py

```diff
@@ -1297,7 +1297,7 @@ class _:
     """
 
 
-@annotate(Param)
+@annotate(Param, cfg = True)
```

Comment:
If we set `cfg = True` for `Param` should we do the same for `ParamSelf`?

Reply:
> Can it happen that a Param in the AST is turned into several CFG nodes?

Currently, no,

> If so, should we also have a CFG node for ClosureExpr?

Doesn't hurt; `ClosureExpr`s don't have any CFG children, though.

Reply:
Actually, we don't want `ClosureExpr`, because we will then incorrectly get a
```ql
ExprCfgNode getBody()
```
predicate, which can never return a value (there is no CFG path from the body to the closure expression itself; they are in different CFG scopes).